benchexec/tools/coveriteam-verifier-validator.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam as coveriteam
+import benchexec.result as result
+import sys
+
+
+class Tool(coveriteam.Tool):
+    """
+    Tool info for a verifier or a validator based on
+    CoVeriTeam: a Configurable Software-Verification Platform.
+    URL: https://gitlab.com/sosy-lab/software/coveriteam.
+    """
+
+    def name(self):
+        return "CoVeriTeam-Verifier-Validator"
+
+    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
+        """
+        Prepare command for the example coveriteam program for a verifier.
+        """
+        # We expect one tasks and a propertyfile.
+        if len(tasks) != 1 or not propertyfile:
+            sys.exit(
```

Comment:
Please split this into two conditions and provide separate error messages. Otherwise this is bad UI if we know precisely what is wrong but let the user guess.

Instead of `sys.exit`, please raise `UnsupportedFeatureException` and please refer to "input files", not "tasks" in the error message.